src/Main.java

```diff
@@ -3,42 +3,59 @@ public static void main(String[] args) {
         System.out.println("Задача №1");
         var dog = 8.0;
+        System.out.println(dog);
         var cat = 3.6;
+        System.out.println(cat);
         var paper = 763789;
+        System.out.println(paper);
+        System.out.println("Задача №2");
+        dog = dog + 4;
         System.out.println(dog);
+        cat = cat + 4;
         System.out.println(cat);
+        paper = paper + 4;
         System.out.println(paper);
-        System.out.println("Задача №2");
-        System.out.println(dog + 4);
-        System.out.println(cat + 4);
-        System.out.println(paper + 4);
         System.out.println("Задача №3");
-        System.out.println(dog - 3.5);
-        System.out.println(cat - 1.6);
-        System.out.println(paper - 7639);
+        dog = dog - 3.5;
+        System.out.println(dog);
+        cat = cat - 1.6;
+        System.out.println(cat);
+        paper = paper - 7639;
+        System.out.println(paper);
         System.out.println("Задача №4");
         var friend = 19;
         System.out.println(friend);
-        System.out.println(friend + 2);
-        System.out.println((friend +2)/7);
+        friend = friend + 2;
+        System.out.println(friend);
+        friend = friend / 7;
+        System.out.println(friend);
         System.out.println("Задача №5");
         var frog = 3.5;
         System.out.println(frog);
-        System.out.println(frog * 10);
-        System.out.println((frog *10)/3.5);
-        System.out.println(((frog * 10)/3.5)+4);
+        frog = frog * 10;
+        System.out.println(frog);
+        frog = frog / 3.5;
+        System.out.println(frog);
+        frog = frog + 4;
+        System.out.println(frog);
         System.out.println("Задача №6");
         var firstBoxer = 78.2;
         var secondBoxer = 82.7;
-        System.out.println(firstBoxer + secondBoxer + " кг.");
-        System.out.println(secondBoxer - firstBoxer + " кг.");
+        var totalWeight = firstBoxer + secondBoxer;
+        System.out.println(totalWeight + " кг.");
+        var massDifference = secondBoxer - firstBoxer;
+        System.out.println(massDifference + " кг.");
         System.out.println("Задача №7");
-        System.out.println(secondBoxer % firstBoxer);
+        var remains = secondBoxer % firstBoxer;
+        System.out.println(remains);
         System.out.println("Задача №8");
         var openingHours = 640;
         var workingHours = 8;
-        System.out.println("Всего работников в компании " + openingHours / workingHours + " человек.");
-        System.out.println("Если в компании работает " + (openingHours / workingHours + 94) + " человек, то всего " + ((openingHours / workingHours + 94) * 8) + " часов работы может быть поделено между сотрудниками.");
+        var workers = openingHours / workingHours;
+        System.out.println("Всего работников в компании " + workers + " человек.");
+        workers = workers + 94;
+        var newOpeningHours = workers * 8;
```

Comment:
```suggestion
        var newOpeningHours = workers * workingHours;
```
числа не используем, используем переменные